src/components/styled/index.tsx

```diff
@@ -34,12 +34,11 @@ export const ServicesCards = styled.div<{ minWidth?: string }>`
 `;
 
 export const PostCardsContainer = styled.div`
-	// TODO make the breakpoint variables
-	@media (min-width: 600px) {
-		display: grid;
-		grid-template-columns: repeat(auto-fit, minmax(336px, 1fr));
-		gap: 4rem;
-	}
+	display: grid;
```

Comment:
Why did you remove the media query?

Reply:
The same design dimensions were needed from small breakpoints. That is why i moved it out of the media query which started at 600px.

Reply:
Okay, cool! 👍🏽